JNIBridge.java

```diff
@@ -14,23 +15,73 @@ static Object newInterfaceProxy(final long ptr, final Class[] interfaces)
 
 	static void disableInterfaceProxy(final Object proxy)
 	{
-		((InterfaceProxy) Proxy.getInvocationHandler(proxy)).disable();
+		if (proxy != null)
+			((InterfaceProxy) Proxy.getInvocationHandler(proxy)).disable();
 	}
 
 	private static class InterfaceProxy implements InvocationHandler
 	{
 		private Object m_InvocationLock = new Object[0];
 		private long m_Ptr;
+		private Constructor<MethodHandles.Lookup> m_constructor;
 
-		public InterfaceProxy(final long ptr) { m_Ptr = ptr; }
+		@SuppressWarnings("unused")
```

Comment:
Is there anything unused in this method?

Reply:
"NoClassDefFoundError e" and "NoSuchMethodException e"